deploy/crds/kieapp.crd.yaml

```
@@ -1913,6 +1913,32 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpmEventEmitters:
+                          description: KafkaJBPMEventEmittersObject kafka configuration
+                            to be used by the KieApp for JBPM Emitter
+                          properties:
+                            acks:
+                              description: Jbpm Emitter, the number of acknowledgments
+                                the producer requires the leader to have received
+                                before considering a request complete, default value
+                                2.
```

There is no default value set for the property.
```suggestion
                                before considering a request complete.
```

---

ok